plugins/inputs/tail/tail.go

```diff
@@ -205,7 +228,20 @@ func (t *Tail) Stop() {
 	t.Lock()
 	defer t.Unlock()
 
+	offsetsMutex.Lock()
+	defer offsetsMutex.Unlock()
+
 	for _, tailer := range t.tailers {
+		if !t.Pipe && !t.FromBeginning {
+			// store offset for resume
+			offset, err := tailer.Tell()
+			if err == nil {
+				offsets[tailer.Filename] = offset
+				log.Printf("D! [inputs.tail] recording offset %d for file: %v", offset, tailer.Filename)
+			} else {
+				t.acc.AddError(fmt.Errorf("E! Error recording offset for file %s\n", tailer.Filename))
```

Comment:
For AddError use error styling: `error recording file offset %s`.

For logging use log style:
```
D! [inputs.tail] Recorded offset %d for file: %s
```

Reply:
I'll fix all of the AddError calls